### PR TITLE
Move things over to meteor

### PR DESCRIPTION
**Pull request by travs**
Still the same skeleton UI, but with a meteor backend instead.
Next step is to update the UI and make this look good.

### CODE REVIEW

File: meteor/imports/api/email-verification.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+Verification.watch = () => {
+  const requestedEvent = poeInstance.Requested({}, {
+    fromBlock: web3.eth.blockNumber,
+    toBlock: 'latest'
+  });
+
+  requestedEvent.watch(Meteor.bindEnvironment((err, evnt) => {
+    if (err) throw err;
```
Comment (retotrinkler) [inline meteor/imports/api/email-verification.js]: think here eventually one should store (or process) incomming events, eg something like https://github.com/melonproject/portal/blob/master/imports/api/orders.js#L48

File: meteor/imports/api/email-verification.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    `Please enter this code at the contract:  ${code}`
+  );
+  var mail = new helper.Mail(from_email, subject, to_email, content);
+
+  return sg.emptyRequest({
+    method: 'POST',
+    path: '/v3/mail/send',
+    body: mail.toJSON()
+  });
+}
```
Comment (retotrinkler) [inline meteor/imports/api/email-verification.js]: like it!

File: meteor/imports/api/modules.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Modules.watch = () => {
+  const registeredEvent = satelliteInstance.ModuleRegistered({}, {
+    fromBlock: web3.eth.blockNumber,
+    toBlock: 'latest'
+  });
+
+  registeredEvent.watch(Meteor.bindEnvironment((err, evnt) => {
+    if (err) throw err;
+    Modules.syncEntryByName(evnt.args.moduleName);
```
Comment (retotrinkler) [inline meteor/imports/api/modules.js]: good

File: meteor/imports/api/modules.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+Modules.syncAll = () => { // can change `fromBlock` if this is slow
+  satelliteInstance.ModuleRegistered({}, {fromBlock: 0, toBlock: 'latest'}).get((err, evntLog) => {
+    if(err) throw err;
+    else if(evntLog.length == 0) console.log('Nothing to sync.');
+    else{
+      for(let evnt of evntLog){
+        Modules.syncEntryByName(evnt.args.moduleName);
+      }
```
Comment (retotrinkler) [inline meteor/imports/api/modules.js]: here eventLog is one such [element](https://github.com/ethereum/wiki/wiki/JavaScript-API#callback-return) right? so you're handling case where more than one module registered per block

File: meteor/imports/api/modules.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    Modules.upsert({
+      name
+    }, {
+      name,
+      owner,
+      url,
+      createdAt: created.toNumber()
+    });
+  });
+};
```
Comment (retotrinkler) [inline meteor/imports/api/modules.js]: nice and modular :)

File: meteor/imports/lib/client/web3.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    console.warn("Using web3 detected from external source.")
+    window.web3 = new Web3(web3.currentProvider);
+  } else {
+    console.warn("No web3 detected. Falling back to http://localhost:8545. You should remove this fallback when you deploy live, as it's inherently insecure. Consider switching to Metamask for development. More info here: http://truffleframework.com/tutorials/truffle-and-metamask");
+    // fallback - use your fallback strategy (local node / hosted node + in-dapp id mgmt / fail)
+    window.web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
+  }
+})
+
+export default window.web3;
```
Comment (retotrinkler) [inline meteor/imports/lib/client/web3.js]: just as an fyi - Simon is working on a redux implemention for this
- https://github.com/melonproject/portal/blob/b715978dcc53ce768a142ede632c2395952c3af4/imports/startup/client/store.js
- https://github.com/melonproject/portal/blob/master/imports/redux/web3.js
- https://github.com/melonproject/portal/blob/master/imports/lib/web3/index.js
think this helped us w metamask

File: meteor/imports/api/email-verification.js (hunk 1 above)
Comment (travs) [inline meteor/imports/api/email-verification.js]: Definitely. I forgot to comment that here, so good job catching it :)
Opens #7

File: meteor/imports/api/modules.js (hunk 4 above)
Comment (travs) [inline meteor/imports/api/modules.js]: Sent you a message about this one.

File: meteor/imports/lib/client/web3.js (hunk 6 above)
Comment (travs) [inline meteor/imports/lib/client/web3.js]: Excellent, will keep this one in mind moving forward.
Opens #8.

**Pull request merged**